### PR TITLE
Cleanup/mixin

### PR DESCRIPTION
**Pull request by Niko-sk2x**
Submitting PR for review. It touches a few parts of code that really should be done right,

### CODE REVIEW

**Comment by Niko-sk2x** [comment]
More commits are to come in this PR, also need to remove the 2 commits that are already in the MC_1.11 branch

File: src/main/java/cubicchunks/asm/mixin/core/client/MixinWorld_HeightLimits.java [hunk 1]
```diff
@@ -59,15 +59,27 @@
...
+    /*
+             if (pos.getY() < this.constant$getLightFromNeighborsFor_getMinHeight$zbd000(0)) {
+                int var10002 = pos.getX();
+                int var10003 = this.constant$getLightFromNeighborsFor_getMinHeight$zbd000(0);
+                int var9 = pos.getZ();
+                int var8 = var10003;
+                pos = new BlockPos(var10002, this.modify$getLightFromNeighborsForGetMinHeight$zbd000(var8), var9);
+            }
+    */
+    // it somehow finds 3 places to modify the constant zero when there are only 2 originally
```
Comment (Foghrye4) [inline src/main/java/cubicchunks/asm/mixin/core/client/MixinWorld_HeightLimits.java]: mixin.out is a decompiled code of a bytecode after injection, yes?

**Comment by Niko-sk2x** [comment]
I will suash commits in this PR, and then add more.

**Comment by Foghrye4** [comment]
I found nothing I should notify you about so far.

**Pull request merged**